server/api/api/endpoints/model_endpoints.py

```diff
@@ -475,25 +494,45 @@ async def get_model_endpoint_monitoring_metrics_values(
 
     :returns:      A list of the results values for this model endpoint.
     """
-    tasks: list[asyncio.Task] = []
+    coroutines: list[typing.Coroutine] = []
```

Comment:
Do you have any reason for this change? I believe it's ok, but tasks may be somewhat faster as they start upon creation

Reply:
[Yup](https://docs.micropython.org/en/latest/library/asyncio.html#asyncio.Task),
> Tasks should not be created directly, rather use create_task to create them.

Could have changed it to `create_task()`, but `asyncio.gather()` does the same thing as the pattern that was there, but is more idiomatic.

It's not correct that tasks start on creation btw. They can only start once the function that created them `await`s or yields in some other way (e.g. with `asyncio.sleep()`).

i.e. if tasks are created in an infinite loop, none of them will ever run.